TestSwitchApi/TestSwitchApi/Repositories/AdminRepo.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Linq;
+using System.Security.Cryptography;
+using Microsoft.AspNetCore.Cryptography.KeyDerivation;
+using TestSwitchApi.Models.ApiModels;
+using TestSwitchApi.Models.DataModels;
+
+namespace TestSwitchApi.Repositories
+{
+    public class AdminRepo : IAdminRepo
+    {
+        private readonly TestSwitchDbContext _context;
+
+        public AdminRepo(TestSwitchDbContext context)
+        {
+            _context = context;
+        }
+
+        public AdminUserDataModel GetAdminByEmail(string email)
+        {
+            return _context.AdminUsers.SingleOrDefault(c => c.Email == email);
+        }
+
+        public AdminUserSession CreateAndStoreSession(int adminUserId)
+        {
+            var newSession = new AdminUserSession
+                { Id = Guid.NewGuid().ToString(), AdminUserID = adminUserId };
+            _context.AdminUserSessions.Add(newSession);
+            _context.SaveChanges();
+            return _context.AdminUserSessions.SingleOrDefault(s => s.Id == newSession.Id);
```

Comment:
I _think_ you can do something like `newSession.Entity` to just get the object straight away???

Reply:
I've tried playing with that and not gotten it working,

Reply:
was I over complicating it?
Is in not just `newSession.Id`?